homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -255,7 +255,7 @@ async def async_step_auth(
             return self.async_abort(
                 reason="no_url_available",
                 description_placeholders={
-                    "docs_url": "https://www.home-assistant.io/docs/configuration/basic/"
+                    "docs_url": "https://www.home-assistant.io/docs/"
```

Comment:
No not the main docs, we need to create a new page dedicated to helping solve this problem. We can make that page a redirect in the future, but it needs to be unique for this error.

Reply:
Maybe even a new `home-assistant.io/error/no-url-available` and have that redirect to `/configuration/basic` for now

Reply:
https://github.com/home-assistant/home-assistant.io/pull/14443